docs/core/integration_system_health.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Integration system health
+sidebar_label: "System health"
+---
+
+The system health platform allows integrations to provide information that helps users understand the state of the integration. This can include details such as the availability of an endpoint, the current server that the integration is connected to, how much of a request quota is still available, etc.
+
+Users can find the aggregated system health by going to **Settings** > **Repairs** and selecting **System information** in the three dots menu.
+
+To provide system health information, the integration should implement `async_register`, which should register the info callback:
+
+```python
+@callback
+def async_register(hass: HomeAssistant, register: system_health.SystemHealthRegistration) -> None:
```

Comment:
I'd include the imports needed from the system health integration, at the top of the module.